action.yml

```
@@ -59,13 +67,36 @@ runs:
       uses: ublue-os/remove-unwanted-software@v6
       if: ${{ inputs.maximize_build_space == 'true' }}
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+      with:
+        install: true
+        cache-binary: ${{ inputs.use_cache }}
```

If we use the `docker` driver we can cut build times in half

```suggestion
        install: true
        driver: docker
        cache-binary: ${{ inputs.use_cache }}
```